package.json

```diff
@@ -91,6 +91,11 @@
     "typescript"
   ],
   "license": "MIT",
+  "lint-staged": {
+    "*.{ts,tsx},!^esm,!^lib": [
```

Comment:
excluded paths `^esm` `^lib`

Reply:
I got the reason why `yarn lint` run successfully because the target files are only under `src` directory. 😄